src/pepr/operator/reconcilers/index.ts

```diff
@@ -12,7 +12,7 @@ const uidSeen = new Set<string>();
  * @param cr The custom resource to check
  * @returns true if the CRD is pending or the current generation has been processed
  */
-export function shouldSkip(cr: UDSExemption | UDSPackage) {
+export function shouldSkip(cr: GenericKind) {
```

Comment:
Should this still be `UDSPackage` instead of `GenericKind`?

Reply:
i went back and forth on that one. I don't think it hurts to set it to `GenericKind` but i'll change it to `UDSPackage`. it'll be more consistent with the other functions that way.